spec/ameba/cli/cmd_spec.cr

```diff
@@ -47,6 +47,11 @@ module Ameba::Cli
         c.rules?.should eq false
       end
 
+      it "defaults skip_reading_config? flag to false" do
+        c = Cli.parse_args %w(file.cr)
+        c.skip_reading_config?.should eq false
```

Comment:
```suggestion
        c.skip_reading_config?.should be_false
```

Reply:
Replace all occurs of `eq false` to `be_false`,  don't know if it's appropriate.